### PR TITLE
MTV 2.2 only Migrating virtual machines

### PR DESCRIPTION
**Pull request by RichardHoch**
MTV 2.2 only. 

Same as the PR Remove "base64-encoded" from source provider instructions #299 https://github.com/konveyor/forklift-documentation/pull/299  **except for comment 1 in step 2**

### CODE REVIEW

File: documentation/modules/migrating-virtual-machines-cli.adoc [hunk 1]
```diff
@@ -51,13 +51,14 @@ type: Opaque
...
+    <{rhv-short}_ca_certificate>
   thumbprint: <vcenter_fingerprint> <4>
 EOF
 ----
-<1> Specify the base64-encoded vCenter user or the {rhv-short} {manager} user.
-<2> Specify the base64-encoded password.
-<3> {rhv-short} only: Specify the base64-encoded CA certificate of the {manager}. You can retrieve it at `https://<www.example.com>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA`.
+<1> Specify the vCenter admin user or the {rhv-short} {manager} user.
+<2> Specify the user password.
+<3> {rhv-short} only: Specify the CA certificate of the {manager}. You can retrieve it at `https://<fqdn>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA`.
```
Comment (mguetta1) [inline documentation/modules/migrating-virtual-machines-cli.adoc:61]: @RichardHoch please remove the "fqdn" change, it is handled in other PR & BZ

File: documentation/modules/migrating-virtual-machines-cli.adoc [hunk 2]
```diff
@@ -79,7 +80,7 @@ spec:
 EOF
 ----
 <1> Allowed values are `ovirt` and `vsphere`.
-<2> Specify the API end point URL, for example, `https://<www.example.com>/sdk` for vSphere or `https://<www.example.com>/ovirt-engine/api/` for {rhv-short}.
+<2> Specify the API end point URL, for example, `https://<fqdn>/sdk` for vSphere or `https://<fqdn>/ovirt-engine/api/` for {rhv-short}.
```
Comment (mguetta1) [inline documentation/modules/migrating-virtual-machines-cli.adoc:83]: @RichardHoch please remove the "fqdn" change, it is handled in other PR & BZ

**Review by mguetta1 — CHANGES_REQUESTED** [review]

**Comment by RichardHoch** [comment]
@mguetta1 -- Please approve this for MTV 2.2.  i removed the "fqdn" changes.

**Review by mguetta1 — APPROVED** [review]
Looks good to me

**Pull request closed without merging**